src/sandstorm/supervisor.c++

```diff
@@ -997,6 +1002,18 @@ void SupervisorMain::setupFilesystem() {
     makeCharDeviceNode("zero", "zero", 1, 5);
     makeCharDeviceNode("random", "urandom", 1, 9);
     makeCharDeviceNode("urandom", "urandom", 1, 9);
+
+    // Create /dev/shm so shm_open() and friends work. Note that even though /dev
+    // is already a tmpfs, we need to mount a separate tmpfs for /dev/shm, because
+    // the former will be read-only.
+    //
+    // TODO: it might be nice to have /dev/shm and /tmp share the same partition,
+    // so we don't have to strictly separate their storage capacity. We could mount
+    // a single tmpfs somewhere invisible, create subdirectories, and then bind-mount
+    // them to their final destinations.
+    KJ_SYSCALL(mkdir("dev/shm", 0700));
```

Comment:
It looks like the `/dev` filesystem itself is created with `0755` permission, any particular reason to use `0700` here? I guess it ultimately doesn't mater since there's only one user in the sandbox and they own everything?